pvlib/pvsystem.py

```diff
@@ -2810,6 +2858,71 @@ def pvwatts_losses(soiling=2, shading=3, snow=0, mismatch=2, wiring=2,
     return losses
 
 
+def dc_ohms_from_percent(V_mp_ref, I_mp_ref, dc_ohmic_percent,
+                         modules_per_string=1,
+                         strings_per_inverter=1):
+    """
+    Calculates the equivalent resistance of the wires from a percent
+    ohmic loss at STC.
+
+    Equivalent resistance is calculated with the function:
+
+    .. math::
+        Rw = (L_{stc} / 100) * (Varray / Iarray)
+
+    :math:`L_{stc}` is the input dc loss as a percent, e.g. 1.5% loss is
+    input as 1.5
+
+    Parameters
+    ----------
+    V_mp_ref: numeric
```

Comment:
Each parameter needs a definition and a unit. E.g.,
```
v_mp_ref: numeric
    Voltage at maximum power in reference conditions [V]
```

I have a slight preference for lower case parameter names here. `V_mp_ref` is the key for the appropriate field in the CEC module database but I don't see that we need to use the exact same name for the function's parameter. The pvsyst module model in pvlib doesn't use that key value, but V_mp_ref appears in the Pvsyst documentation. SAPM calls it `Vmpo`.

Reply:
Added definitions and units and changed to lowercase generic parameters vmp_ref and imp_ref.